pkg/psmdb/mongos.go

```diff
@@ -255,6 +255,11 @@ func mongosContainerArgs(cr *api.PerconaServerMongoDB, useConfigFile bool, cfgIn
 	}
 
 	if cr.TLSEnabled() {
+		if !*cr.Spec.TLS.AllowInvalidCertificates {
```

Comment:
Can we add e2e test case for it? We need to be sure that we do not lose any key from the command.